src/Datagrid/PagerInterface.php

```diff
@@ -26,6 +26,7 @@
  * @method bool                     isFirstPage()
  * @method bool                     isLastPage()
  * @method int                      getNbResults()
```

Comment:
You can remove this

Reply:
technically we still need it? We are adding `countResults()` and deprecating `getNbResults()`. We are not removing `getNbResults()` yet?

Reply:
I mean who knows what depends on `getNbResults()` out there on `3.x` :smile:

Reply:
So its still part of the current contract as I see it

Reply:
ok